src/ui/common/src/components/pages/workflows/components/CheckItem.tsx

```diff
@@ -17,23 +17,23 @@ import { ShowMore } from './MetricItem';
 
 const errorIcon = (
```

Comment:
Can we actually refactor these to use a StatusIndicator component with a height of 20px?

Then inside the StatusIndicator Component, we can have the colors set in there to theme.palette.SomeStatus

Reply:
We cannot completely replace the Check icons with StatusIndicator because if status==succeeded, depending on the value of the check, it has different displays. I refactored CheckItem & MetricItem to use StatusIndicator for all cases that aren't status==succeeded.

Let me know if you think I should make CheckStatusIndicator/MetricStatusIndicator components or if stick with this.

Reply:
okay, should be fine for now then.